### PR TITLE
Add mail service

### PR DESCRIPTION
**Pull request by Porquepix**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -25,3 +25,8 @@ export DATABASE_SSL=false
 export DATABASE_URL="postgres://postgres:root@localhost:5434/dev_prello"
 
 export TEST_DATABASE_URL="postgres://postgres:root@localhost:5434/test_prello"
+
+export SMTP_USER=""
```
Comment (YvesLongchamp) [inline .env.dev:29]: SMTP ?

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "node": "8.3.0"
   },
   "scripts": {
-    "start": "node dist/server.js",
+    "start": "node dist/src/server.js",
```
Comment (YvesLongchamp) [inline package.json:15]: Why a src/ folder in dist ?

**Review by YvesLongchamp — COMMENTED** [review]
Good for me, just need explanations for the two comments

File: .env.dev (hunk 1 above)
Comment (Porquepix) [reply .env.dev:29]: https://fr.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol

Comment (Porquepix) [reply .env.dev:29]: It is the protocol used by the lib.

**Review by YvesLongchamp — APPROVED** [review]

**Pull request merged**